src/main/java/org/eclipse/tractusx/ssi/lib/exception/proof/JwtAudienceCheckFailedException.java

```diff
@@ -19,12 +19,14 @@
  * *******************************************************************************
  */
 
-package org.eclipse.tractusx.ssi.lib.exception;
+package org.eclipse.tractusx.ssi.lib.exception.proof;
 
 import java.util.List;
+import org.eclipse.tractusx.ssi.lib.exception.SSIException;
 
 /** The type Jwt audience check failed exception. */
-public class JwtAudienceCheckFailedException extends JwtException {
+public class JwtAudienceCheckFailedException extends SSIException {
```

Comment:
Think about renaming this class to e.g. **JwtInvalidAudienceException**. Term like "failed" / "check" should not be included in class names.